src/mersenne.ts

```diff
@@ -20,14 +20,14 @@ export class Mersenne {
   }
 
   /**
-   * Generates a random number between `[min, max)`.
+   * Generates a random number between `[max, min)`.
```

Comment:
```suggestion
   * Generates a random number between `[min, max)`.
```
I think this should be changed back to what it was before (as seen in the suggestion block). The original description said that the function generates a number with an inclusive lower bound of ```min``` and an exclusive upper bound of ```max```, so this change will change the meaning of the JSDoc. Also, we should probably discuss putting a "max is first param" text in the JSDoc, but I'm not sure what format that would take.

Reply:
I've been thinking about this too, which one should we change to clear out the confusion. Is it the docs that's inconsistent with the implemenetation of the param, or is it the param which might be better if we put min first instead of max. 

Maybe it's better to fix the param and keep the docs as it is before, but I'm not sure about this. Wdyt? 🤔

Reply:
I think fixing the param itself would be a breaking API change, so we should just change the doc. @Shinigami92 @ST-DDT @pkuczynski What do you think?

Reply:
Yes, this would be a breaking change so quite problematic...

Reply:
For now, please only the jsdocs please.